_posts/2017-02-06-on-web-typography-review.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title: On Web Typography - a Book Review
+date: 2017-02-06
+author: Anne Tomasevich
+tags: design typography
+featured_image: "/blog/on-web-typography.jpg"
+featured_image_alt: "Copies of the book On Web Typography"
+featured_image_caption: "*[Image source](https://abookapart.com/products/on-web-typography)*"
+featured_image_height: "917px"
+featured_image_width: "1474px"
+summary: |
+  A review of Jason Santa Maria's *On Web Typography*, and how we applied the
+  lessons learned to our workflow.
+---
+
+As a developer who often interacts with and creates web designs, I'm always on
+the lookout for quality resources to level up my design skills, and the [Book
+Apart series](https://abookapart.com/) has proven to be an excellent source of
+knowledge presented in a relevant, concise manner. My latest read was [*On Web
+Typography* by Jason Santa Maria](https://abookapart.com/products/on-web-typography),
+and I found the book to be a strong resource, especially for the non-designer.
+In this post, I'll review the book and detail how we at Savas Labs implemented
+the lessons we learned.
+
+## Thinking about reading
+
+Have you ever stopped to think about the actual process of reading? I think
+Santa Maria puts it best:
+
+> Reading is something we do every day, but we can easily take it for granted. Slapping words on a page won’t ensure good communication, just as mashing your hands across a piano won’t make for a pleasant composition. The experience of reading and the effectiveness of our message are determined by both what we say and how we say it.
+
+Rather than jumping into choosing and pairing typefaces, or even evaluating
+their characteristics, Santa Maria suggests we approach typography much further
+back and take the time to consider what users are actually doing when they read
+text on our sites. This is something I hadn't considered before and brings a
+depth to the process that is so much more than picking and applying fonts.
+
+## Base knowledge
+
+Most of us can tell a serif from a sans serif, but I found the base of knowledge
+Santa Maria provides in the book to be an incredibly useful foundation. Knowing
```

Comment:
`...the base of knowledge Santa Maria provides` - it may be worth rewording this part to `the "Base of Knowledge" section Santa M...` to communicate that the base of knowledge is an actual section in the book. You'll know better if it makes sense or not.